website/docs/addon_applications_admin.md

```diff
@@ -1,63 +1,294 @@
 ---
 id: addon_applications_admin
 title: Applications and Tools
-sidebar_label: Applications addon settings
+sidebar_label: Applications and Tools
+description: Applications Addon Admin Documentation
+toc_max_heading_level: 5
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 
+<ReactMarkdown>
+{versions.Applications_Badge}
+</ReactMarkdown>
 
-## Applications
+## Introduction
 
-In this section you can manage what Applications are available to your studio, locations of their executables, and their additional environments. In AYON context, each application that is integrated is also called a `Host` and these two terms will be used interchangeably in the documentation.
+The Applications addon is designed to configure applications, set their executable paths, and set up their startup environments.
 
-Each Host is made of two levels.
-1. **Application group** - This is the main name of the application and you can define extra environments that are applicable to all versions of the given application. For example any extra Maya scripts that are not
-version dependent, can be added to `Maya` environment here.
-2. **Application versions** - Here you can define executables (per platform) for each supported version of
-the DCC and any default arguments (`--nukex` for instance). You can also further extend its environment.
+<!-- :::info
+The addon introduces a debug launcher action for users. For more details, please refer to [Application addon artist docs] (addon_applications_artist.md)
+Also, The addon's client code features hook definitions that developers can leverage to implement launch hooks that tailor launch processes for other addons. For more details, please see [Application addon Dev docs] (addon_applications_dev.md).
+::: -->
```

Comment:
since, https://github.com/ynput/ayon-applications/pull/12 is merged. 
shall we uncomment this section ? 

```suggestion
:::info
The addon introduces a debug launcher action for users. For more details, please refer to [Application addon artist docs] (addon_applications_artist.md)
:::
```

also, should we have a page for applications addon artist ?

Reply:
I removed it.
https://github.com/ynput/ayon-documentation/pull/220/commits/d7205537ed64af341d102c0e94c5e5c5e394b32b